src/components/Exercises.js

```diff
@@ -14,11 +14,12 @@ const Exercises = ()=>{
   useEffect(()=>{
     const getExercises = async ()=>{
       try {
-        const res = await axios.get('/api/exercise')
+        const res = await axios.get('/api/exercise/userExercises')
         setExercisesData((exercisesData)=>({
           ...exercisesData,
           exercises: res.data
         }))
+        console.log(exercisesData.exercises.length)
```

Comment:
I would remove any console.log messages like this that are mostly for debugging.